hnn_core/feed.py

```diff
@@ -126,15 +228,16 @@ def feed_event_times(feed_type, target_cell_type, params, gid):
             numspikes=50,
             prng=prng)
     elif feed_type == 'common' and not all_syn_weights_zero:
-        event_times = _create_common_input(
+        event_times = _create_bursty_input(
             distribution=params['distribution'],
             t0=params['t0'],
             t0_stdev=params['t0_stdev'],
             tstop=params['tstop'],
             f_input=params['f_input'],
-            stdev=params['stdev'],
+            events_jitter_std=params['stdev'],
```

Comment:
Is "jitter" necessary for the variable name? While it gives an intuitive sense for its purpose in the end it's still just standard deviation.

Reply:
See also my [comment above](https://github.com/jonescompneurolab/hnn-core/pull/221/files#r551677882).